src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -168,7 +169,9 @@ class DropdownMenu extends React.Component<Props, State> {
             !menuEl.contains(event.target) &&
             !menuButtonEl.contains(event.target)
         ) {
+            const { onMenuClose = noop } = this.props;
```

Comment:
Perhaps we could move this to `defaultProps` and then simplify `handleMenuClose` too?

Reply:
Great idea. Will fix

Reply:
Seems like defaulting to noop in defaultprops should be good enough? Is the default when destructuring needed?

Reply:
I ran into a bunch of Flow issues when I tried to add `onMenuClose` as a default prop while also keeping it optional, so I resorted to destructuring instead.